{
  "type": "issue",
  "issue": {
    "id": 301076499,
    "node_id": "MDU6SXNzdWUzMDEwNzY0OTk=",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/12566",
    "number": 12566,
    "state": "closed",
    "state_reason": "completed",
    "title": "[0.16]: Compiling warnings while compiling bitcoin-qt, tx ...",
    "body": "Hello,\r\n\r\nWhile I'm compiling Bitcoin on my 16.04 Ubuntu, I get some warnings.\r\nIt seems, that while all executable files where compiled, I get these errors.\r\n\r\nYou can see a full output in the .txt file.\r\n\r\nRegards.\r\n[output-btc.txt](https://github.com/bitcoin/bitcoin/files/1767772/output-btc.txt)\r\n",
    "user": {
      "login": "globaltoken",
      "id": 24831016,
      "node_id": "MDQ6VXNlcjI0ODMxMDE2",
      "avatar_url": "https://avatars.githubusercontent.com/u/24831016?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/globaltoken",
      "html_url": "https://github.com/globaltoken",
      "followers_url": "https://api.github.com/users/globaltoken/followers",
      "following_url": "https://api.github.com/users/globaltoken/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/globaltoken/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/globaltoken/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/globaltoken/subscriptions",
      "organizations_url": "https://api.github.com/users/globaltoken/orgs",
      "repos_url": "https://api.github.com/users/globaltoken/repos",
      "events_url": "https://api.github.com/users/globaltoken/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/globaltoken/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [],
    "assignees": [],
    "author_association": "NONE",
    "locked": true,
    "active_lock_reason": "resolved",
    "comments": 3,
    "closed_at": "2018-03-14T14:22:34Z",
    "created_at": "2018-02-28T15:50:17Z",
    "updated_at": "2021-09-08T12:28:14Z"
  },
  "events": [
    {
      "event": "commented",
      "id": 369315486,
      "node_id": "MDEyOklzc3VlQ29tbWVudDM2OTMxNTQ4Ng==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/369315486",
      "actor": {
        "login": "randolf",
        "id": 33541170,
        "node_id": "MDQ6VXNlcjMzNTQxMTcw",
        "avatar_url": "https://avatars.githubusercontent.com/u/33541170?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/randolf",
        "html_url": "https://github.com/randolf",
        "followers_url": "https://api.github.com/users/randolf/followers",
        "following_url": "https://api.github.com/users/randolf/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/randolf/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/randolf/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/randolf/subscriptions",
        "organizations_url": "https://api.github.com/users/randolf/orgs",
        "repos_url": "https://api.github.com/users/randolf/repos",
        "events_url": "https://api.github.com/users/randolf/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/randolf/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-02-28T17:27:13Z",
      "updated_at": "2018-02-28T17:27:13Z",
      "author_association": "CONTRIBUTOR",
      "body": "This looks like a problem with Boost libraries.  Maybe a version mismatch?  I've had problems with Boost in the past where they broke backward compatibility.\r\n\r\nDoes the compiled code seem to execute correctly?",
      "user": {
        "login": "randolf",
        "id": 33541170,
        "node_id": "MDQ6VXNlcjMzNTQxMTcw",
        "avatar_url": "https://avatars.githubusercontent.com/u/33541170?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/randolf",
        "html_url": "https://github.com/randolf",
        "followers_url": "https://api.github.com/users/randolf/followers",
        "following_url": "https://api.github.com/users/randolf/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/randolf/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/randolf/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/randolf/subscriptions",
        "organizations_url": "https://api.github.com/users/randolf/orgs",
        "repos_url": "https://api.github.com/users/randolf/repos",
        "events_url": "https://api.github.com/users/randolf/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/randolf/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12566#issuecomment-369315486",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566"
    },
    {
      "event": "commented",
      "id": 369331313,
      "node_id": "MDEyOklzc3VlQ29tbWVudDM2OTMzMTMxMw==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/369331313",
      "actor": {
        "login": "globaltoken",
        "id": 24831016,
        "node_id": "MDQ6VXNlcjI0ODMxMDE2",
        "avatar_url": "https://avatars.githubusercontent.com/u/24831016?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/globaltoken",
        "html_url": "https://github.com/globaltoken",
        "followers_url": "https://api.github.com/users/globaltoken/followers",
        "following_url": "https://api.github.com/users/globaltoken/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/globaltoken/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/globaltoken/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/globaltoken/subscriptions",
        "organizations_url": "https://api.github.com/users/globaltoken/orgs",
        "repos_url": "https://api.github.com/users/globaltoken/repos",
        "events_url": "https://api.github.com/users/globaltoken/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/globaltoken/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-02-28T18:14:50Z",
      "updated_at": "2018-02-28T18:14:50Z",
      "author_association": "NONE",
      "body": "Yes, the Wallet works fine, and this are just warnings for the Code.\r\nBut in older Versions like 0.15 I never received such warnings.\r\n",
      "user": {
        "login": "globaltoken",
        "id": 24831016,
        "node_id": "MDQ6VXNlcjI0ODMxMDE2",
        "avatar_url": "https://avatars.githubusercontent.com/u/24831016?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/globaltoken",
        "html_url": "https://github.com/globaltoken",
        "followers_url": "https://api.github.com/users/globaltoken/followers",
        "following_url": "https://api.github.com/users/globaltoken/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/globaltoken/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/globaltoken/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/globaltoken/subscriptions",
        "organizations_url": "https://api.github.com/users/globaltoken/orgs",
        "repos_url": "https://api.github.com/users/globaltoken/repos",
        "events_url": "https://api.github.com/users/globaltoken/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/globaltoken/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12566#issuecomment-369331313",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566"
    },
    {
      "event": "commented",
      "id": 373038374,
      "node_id": "MDEyOklzc3VlQ29tbWVudDM3MzAzODM3NA==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/373038374",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-14T14:22:34Z",
      "updated_at": "2018-03-14T14:22:34Z",
      "author_association": "MEMBER",
      "body": "These and similar warnings have been appearing in gitian builds for some time.",
      "user": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12566#issuecomment-373038374",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12566"
    },
    {
      "event": "closed",
      "id": 1521058639,
      "node_id": "MDExOkNsb3NlZEV2ZW50MTUyMTA1ODYzOQ==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/1521058639",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-14T14:22:34Z"
    },
    {
      "event": "locked",
      "id": 5271980084,
      "node_id": "LOE_lADOABII584R8hATzwAAAAE6PAg0",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/5271980084",
      "actor": {
        "login": "MarcoFalke",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/MarcoFalke",
        "html_url": "https://github.com/MarcoFalke",
        "followers_url": "https://api.github.com/users/MarcoFalke/followers",
        "following_url": "https://api.github.com/users/MarcoFalke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/MarcoFalke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/MarcoFalke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/MarcoFalke/subscriptions",
        "organizations_url": "https://api.github.com/users/MarcoFalke/orgs",
        "repos_url": "https://api.github.com/users/MarcoFalke/repos",
        "events_url": "https://api.github.com/users/MarcoFalke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/MarcoFalke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2021-09-08T12:28:14Z",
      "lock_reason": "resolved"
    }
  ]
}